File: fizzbuzz/src/kotlin/fizzbuzz.kt
fun main() {
    for (n in 1..50) {
        when {
            n % 3 == 0 && n % 5 == 0 -> println("fizz buzz")
            n % 3 == 0 -> println("fizz")
            n % 5 == 0 -> println("buzz")
            else -> println(n)
        }
    }
}

File: fizzbuzz/src/clojure/fizzbuzz.clj
(doseq [n (range 1 51)] 
  (println
    (cond
      (and (= (mod n 3) 0) (= (mod n 5) 0)) "fizz buzz"
      (= (mod n 3) 0) "fizz"
      (= (mod n 5) 0) "buzz"
      :else (str n))))
